cron_job_definitions.go

```diff
@@ -30,7 +28,7 @@ func taskCleanupDraftTransactions(ctx context.Context, logClient zLogger.GormLog
 
 	// Get the records
 	if err := getModels(
-		ctx, NewBaseModel(ModelNameEmpty, opts...).Client().Datastore(),
+		ctx, NewBaseModel(ModelNameEmpty, WithClient(client)).Client().Datastore(),
```

Comment:
Please replace `NewBaseModel(ModelNameEmpty, WithClient(client)).Client().Datastore()` with `client.Datastore()`

Reply:
Thanks for noticing that. I was definitely over-complicated code. Done.